crates/unified-bridge/src/imported_bridge_exit.rs

```diff
@@ -285,30 +291,33 @@ pub struct ImportedBridgeExit {
     pub claim_data: Claim,
 
     /// The global index of the imported bridge exit.
-    pub global_index: GlobalIndex,
+    pub global_index: U256,
```

Comment:
Also, in the spirit of the recent changes moving us towards stronger types, it would be nicer to have some sort of `RawGlobalIndex` type wrapper here instead of plain `U256`.